Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -137,20 +137,19 @@ public override void OnPreSceneQuery()
             {
                 Vector3 endPoint = Vector3.forward * SphereCastRadius;
                 Rays[0].UpdateRayStep(ref pointerPosition, ref endPoint);
+                PrioritizedLayerMasksOverride = PrioritizedLayerMasksOverride ?? GrabLayerMasks;
```

Comment:
This line is making me think. IIf somebody sets this to some custom value to override, and then to reset the behavior, sets PrioritizedLayerMasksOverride to null, are there cases where GrabLayerMasks would not properly be used? I *think* it's okay but the line is giving me some pause.

Reply:
I think this should be fine unless the user manages to set PrioritizedLayerMasksOverride to null between OnPreSceneQuery and the QueryScene calls

Reply:
Sorry for getting to this late - I think that the chance for issues here is pretty minimal as called out, but it's still a case where someone's public surface modification (i.e. someone telling us to set a property) can get wiped out by something that shouldn't be wiping them out.

If we're concerned about doing this reconciliation on each pass, the right thing to do would be to modify the property setter or have some other one-time process of reconciliation - this is kinda an awkward in between words where we're actually doing reconciliation each time (and doing it on a property that affects data set by callers).